src/aiortc/contrib/getkeypoints.py

```diff
@@ -18,20 +18,8 @@ def __init__(self):
                                                flip_input=True, device='cpu')
 
     def get_keypoints(self, input_frames, image_size = 256, crop_data = False):
-        """Generates dataset images, keypoints (also called poses),
+        """ Generates dataset images, keypoints (also called poses),
          and segmenatations from input_frames/ frames
```

Comment:
very minor suggestion: would shift the "and" to match the place where Generates starts, but this is truly a personal style.

Reply:
done